crates/orderbook/src/orderbook.rs

```diff
@@ -281,7 +302,7 @@ impl Orderbook {
 
         for order in &orders {
             tracing::debug!(order_uid =% order.metadata.uid, "order cancelled");
-            Metrics::on_order_operation(order, OrderOperation::Cancelled);
+            Metrics::on_order_operation(order, OrderOperation::Cancelled, None);
```

Comment:
I am not so convinced about setting this to `None`. So I want to flag it out to start a discussion if you guys agree. We could fetch the quote from the database :thinking:

Reply:
Setting this to `None` will result in all cancelled orders being counted as `limit` orders which does not seem great...

Reply:
Perfect! I will address it 👌